Economy/Data/Scripts/Economy.scripts/EconConfig/EconDataManager.cs

```diff
@@ -993,8 +997,6 @@ public static void CreatePlayerMarket(ulong accountId, long entityId, double siz
                         item.IsBlacklisted = true;
                 }
             }
-
-            EconomyScript.Instance.Data.Markets.Add(market);
```

Comment:
hmm probably should have pointed this out before accepting the request, but the summary page wasnt obvious about this change for some reason.

Unless you changed how this works - wont removing this prevent newly detected items actually being added to the market table?

Reply:
I call that later just refactoring